robot_smach_states/src/robot_smach_states/util/designators/utility.py

```diff
@@ -8,10 +8,28 @@
 import operator
 
 # TU/e Robotics
-from .core import Designator, VariableDesignator
+from robot_smach_states.util.designators.core import Designator, VariableDesignator
```

Comment:
Why??

Reply:
My Python install complained about this not happening in a package.

Reply:
Weird